client/src/pages/Login.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import logo from "../components/images/FSAE-Logo.png"
+
+function Login() {
+    return (
+        <div class= "sign-in-h">
+            <img className= "logo-signin" src={logo}/>
+            <h1 class= "sign-in">Sign in</h1>
+            <form class= "sign-in-body">
+               
+                <label for="email">CSUN email:</label><br/>
+                <input type="text" id="email" name="email" /><br/>
+                <label for="password">Password:</label><br/>
+                <input type="text" id="password" name="password" /><br/>
+        
+                <input type="submit" value="Submit" />
+            </form>
+           
+        </div>  
+    )
+}
+
+export default Login;
```

Comment:
Login.js page should be removed.  We will not have a login page since it will be handled in the header and we are using OAuth 2.0 to authenticate with google.

Reply:
check update